swag-chart/values.yaml

```diff
@@ -1,8 +1,9 @@
 accounts:
+  app: swag-shop
```

Comment:
why not one setting of app that they all use? i wouldn't block the PR just for this though.

Reply:
I'm using a feature in Helm where the scope inside each manifest file is made to get values from its respective variables inside values.yaml.

I get why you would be concerned about this. Do you think changing the app value to be hardcoded for each manifest would be valid solution? App label is meant to differentiate deployments/services which are a part of the swag-app from non-swag-apps. So I don't see why I shouldn't hardcode it.

Reply:
just because you're using that `with` scope shouldn't mean you can't also access other values outside that scope. you could just move the app value out of each service-specific mapping to one common place.